anthos-bm-apigee/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Apigee Hybrid on Anthos Baremetal on Google Compute Engine VMs with Terraform
+
+This repository is a sample Apigee Hybrid installation on Anthos Cluster on bare metal running on GCE Virtual Machine. This also assumes that Apigee organization name is same as the GCP Project Id. The installation will also create a trial Apigee Edge organization with same name as project id if  Apigee Organization is not available.
+
+### Pre-requisites
+
+- A workstation with access to internet _(i.e. Google Cloud APIs)_ with the following installed
+  - [Git](https://www.atlassian.com/git/tutorials/install-git)
+  - [Google Cloud SDK](https://cloud.google.com/sdk/docs/install)
+  - [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) (>= v0.15.5, < v1.2)
+
+- A [Google Cloud Project](https://console.cloud.google.com/cloud-resource-manager?_ga=2.187862184.1029435410.1614837439-1338907320.1614299892) _(in which the resources for the setup will be provisioned)_
+
+- Enable Compute Engine API Services and Apigee API Services.
```

Comment:
Can we add this as a pre-step to the Terraform script itself, without asking them to enable it?

Reply:
Sometimes these are problem with argolis environment.  In argolis the default compute service account created doesn't add Editor role and needs to be manually added. This is one of the main reason why even the base scripts fails in argolis.

Reply:
Yes we can remove it.

Reply:
Removed this part but kept the ensure part.